nextstrain/cli/runner/aws_batch/s3.py

```diff
@@ -65,6 +65,7 @@ def upload_workdir(workdir: Path, bucket: S3Bucket, run_id: str) -> S3Object:
     ])
 
     # Stream writes directly to the remote ZIP file
+    remote_file: Any
```

Comment:
In the future, we may consider creating [type stubs](https://github.com/microsoft/pyright/blob/main/docs/type-stubs.md) for `fsspec` so that we can provide the correct type instead of `Any`.